src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,243 @@
+                                {this.state.pronouns === CONST.PRONOUNS.SELF_SELECT && (
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.selfSelectedPronouns}
+                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
+                                        placeholder="Self-select your pronoun"
+                                    />
+                                )}
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>
+                                    {Str.isSMSLogin(this.props.myPersonalDetails.login)
+                                        ? 'Phone Number' : 'Email Address'}
+                                </Text>
+                                <TextInput
+                                    style={[styles.textInput, styles.disabledTextInput]}
+                                    value={this.props.myPersonalDetails.login}
+                                    editable={false}
+                                />
+                            </View>
+                            <View style={styles.mb2}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Timezone</Text>
+                                <RNPickerSelect
+                                    onValueChange={selectedTimezone => this.setState({selectedTimezone})}
+                                    items={timezones}
+                                    style={this.state.isAutomaticTimezone ? {
+                                        ...styles.picker,
+                                        inputIOS: [styles.picker.inputIOS, styles.textInput, styles.disabledTextInput],
+                                        inputAndroid: [
+                                            styles.picker.inputAndroid, styles.textInput, styles.disabledTextInput,
+                                        ],
+                                        inputWeb: [styles.picker.inputWeb, styles.textInput, styles.disabledTextInput],
+                                    } : styles.picker}
+                                    useNativeAndroidPickerStyle={false}
+                                    value={this.state.selectedTimezone}
+                                    Icon={() => <Icon src={DownArrow} />}
+                                    disabled={this.state.isAutomaticTimezone}
+                                />
+                            </View>
+                            <Checkbox
+                                label="Set my timezone automatically"
+                                isChecked={this.state.isAutomaticTimezone}
+                                onCheckboxClick={isAutomaticTimezone => this.setState({isAutomaticTimezone})}
```

Comment:
If the user checks `Set my timezone automatically` we should use `moment.tz.guess()` to get their current timezone and update the dropdown as well.

ex. from E.com

https://user-images.githubusercontent.com/16747822/111397889-94c77400-867f-11eb-944d-3439d110698b.mov